### ISSUE TITLE
Output Mode: Mouse - Gyro calibration button missing, and calibration not working

### ISSUE THREAD

**Issue by AdriandeLima** [Ryochan7/DS4Windows#1878]
**Describe the bug**
Reopening issue #1859 (sorry 😢) - Gyro Calibration button is still missing from Output Mode: Mouse.

Gyro Calibration function is now also broken in Output Mode: Mouse.

**v2.2.4** - Gyro Calibration button is present only on one Output Mode screen (I can't remember which one). Clicking it calibrated my drifting gyro, and applied the calibration for all output modes, including Output Mode: Mouse. It worked perfectly, though it was mildly annoying to have to go to another Output Mode to calibrate, before I could set up my controls on my desired Output Mode.

**v2.2.6** - Gyro Calibration button is present in all output modes except Output Mode: Mouse. Clicking it seems to calibrate the gyro for the respective Output Mode; since the button is missing from Output Mode: Mouse, I cannot calibrate my gyro for mouse controls, and it drifts downwards (problem with the controller).

**Expected behavior**
Please could you add Gyro Calibration button/function to Output Mode: Mouse? I'd much appreciate it!

**Desktop (please complete the following information):**
 - Controller Make and Model: PicTek HD-060/PC263 with a gyro drift problem
 - OS: Win10 v1909 Pro
 - DS4Windows Version: v2.2.6 (last confirmed working v2.2.4)

**Comment by Ryochan7** [comment]
Fixed in version 2.2.7

**Comment by AdriandeLima** [comment]
@Ryochan7 Thank you so much for adding the button!

Unfortunately, v2.2.4 is still the last version where Gyro Calibration works for Output Mode: Mouse for me. v2.2.5 up to v2.2.7, even though they appear to calibrate, still result in the gyro drifting constantly down.

I have attached the profile I use for testing (originally created in v2.2.4, imported via auto-update into v2.2.7, then I dropped it back into v2.2.4), and the game I use is [Krunker.io](https://krunker.io/
[Gyro FPS.txt](https://github.com/Ryochan7/DS4Windows/files/6009392/Gyro.FPS.txt)

)
